qa/config/crimson_seastore.yaml

```diff
@@ -0,0 +1,20 @@
+overrides:
+  ceph:
+    fs: xfs
```

Comment:
What is `fs: xfs` used for?

Reply:
I...think that might be a holdover from filestore, the predecessor to bluestore which needed an actual filesystem.  The only logic I see in ceph.py that actually checks that variable causes some xfs packages to be installed.  That fs: xfs line seems to exist throughout the qa/suites directory and should probably be cleaned up generally.